File: 6KYU/Create Phone Number/solution.go
package kata

import (
	"fmt"
)

func CreatePhoneNumber(numbers [10]uint) string {
	a := ""
	b := ""
	c := ""

	for i := 0; i < 3; i++ {
		a += fmt.Sprint(numbers[i])
	}

	for i := 3; i < 6; i++ {
		b += fmt.Sprint(numbers[i])
	}

	for i := 6; i < len(numbers); i++ {
		c += fmt.Sprint(numbers[i])
	}

	return fmt.Sprintf("(%s) %s-%s", a, b, c)
}

File: 6KYU/Create Phone Number/solution.cs
using System.Linq;

public class Kata
{
  public static string CreatePhoneNumber(int[] numbers)
  {
      var a = string.Join("", numbers.Take(3));
      var b = string.Join("", numbers.Skip(3).Take(3));
      var c = string.Join("", numbers.Skip(6));
      return $"({a}) {b}-{c}";
  }
}